src/csharp/Grpc.Tools.MSBuildIntegration/Grpc.Tools.MSBuildIntegration.targets

```diff
@@ -0,0 +1,27 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">

```

Comment:
Does intellisense code completion work well if when this nuget package is used?
#4805 contains ideas how to achieve that.